### PR TITLE
Do no export file field in XMP

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes https://github.com/JabRef/jabref-issue-melting-pot/issues/271

### Mandatory checks

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/logic/xmp/XmpUtilWriter.java [hunk 1]
```diff
@@ -237,34 +235,42 @@ private void writeDocumentInformation(PDDocument document,
...
+                        di.setTitle(value);
+                case StandardField.KEYWORDS ->
+                        di.setKeywords(value);
+                case StandardField.ABSTRACT ->
+                        di.setSubject(value);
+                case StandardField.FILE -> {
+                    // do not write file field
+                }
+                case null, default ->
+                        resolvedEntry.getField(field).ifPresent(val -> di.setCustomMetadataValue("bibtex/" + field, val));
```
Comment (koppor) [inline src/main/java/org/jabref/logic/xmp/XmpUtilWriter.java:272]: Move one line down to keep the comment above the statement

File: src/test/java/org/jabref/logic/exporter/XmpPdfExporterTest.java [hunk 2]
```diff
@@ -61,46 +61,46 @@ class XmpPdfExporterTest {
...
-        olly2018.setField(StandardField.COMMENT, "COMMENT");
-        olly2018.setField(StandardField.DOI, "10/3212.3123");
-        olly2018.setField(StandardField.FILE, ":article_dublinCore.pdf:PDF");
-        olly2018.setField(StandardField.GROUPS, "NO");
-        olly2018.setField(StandardField.HOWPUBLISHED, "online");
-        olly2018.setField(StandardField.KEYWORDS, "k1, k2");
-        olly2018.setField(StandardField.OWNER, "me");
-        olly2018.setField(StandardField.REVIEW, "review");
-        olly2018.setField(StandardField.URL, "https://www.olly2018.edu");
+        OLLY_2018.setCitationKey("Olly2018");
```
Comment (koppor) [inline src/test/java/org/jabref/logic/exporter/XmpPdfExporterTest.java:64]: Can we do withCitationKey and move the initialization to the variable?

**Review by koppor — COMMENTED** [review]
Nice one! Small comments 😅

File: src/test/java/org/jabref/logic/exporter/XmpPdfExporterTest.java (hunk 2 above)
Comment (Siedlerchr) [reply src/test/java/org/jabref/logic/exporter/XmpPdfExporterTest.java]: Yes, but moving the initialization to the variable is a bad idea due to how the test works. Due to MethodSources it has to be static and we are setting the vars to the correct values and we are resetting them again.

**Review by koppor — APPROVED** [review]

**Pull request merged**